src/http/register-websocket/upgrade.js

```diff
@@ -16,14 +17,26 @@ module.exports = function upgrade (wss, { cwd, inventory, update }) {
     let lambda = get.ws('connect')
 
     // Create a connectionId uuid
```

Comment:
Woops, this is probably my comment – this does not generate `uuid`s per spec, but simply GUIDs. Feel free to change (or I can later!)